### PR TITLE
Update branding and temporarily disable the Consortium section

### PR DESCRIPTION
**Pull request by pavel-kirienko**
@thirtytwobits this is my first stab toward rebranding. This changeset replaces UAVCAN with Cyphal and OpenCyphal (I was careful with their differentiation but may have slipped up once or twice) and temporarily deactivates the Consortium section (it is still there but not linked from anywhere).

@clyde-johnston the `uat` branch did not get deployed to http://uat.uavcan.org:8080/, can you please check?

### CODE REVIEW

**Comment by clyde-johnston** [comment]
@pavel-kirienko It deployed successfully but the UAT site was disabled in Apache. I have re-enabled it and reloaded the server.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,13 @@
-# UAVCAN website
+# Cyphal website
 
-Sources of the [uavcan.org](https://uavcan.org) website.
-
-If you want your company to be listed on the front page,
-[apply to become a member of the UAVCAN Consortium](https://uavcan.org/consortium).
+Sources of the [cyphal.org](https://cyphal.org) website (formerly uavcan.org).
```
Comment (thirtytwobits) [inline README.md]: opencyphal.org?

File: app/templates/base/head.html [hunk 2]
```diff
@@ -10,16 +10,17 @@
...
 <meta name="theme-color" content="#1700b3">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-<meta name="author" content="UAVCAN Development Team">
+<meta name="author" content="OpenCyphal Development Team">
 
 <meta property="og:type"      content="website">
-<meta property="og:site_name" content="Uncomplicated Application-layer Vehicular Computing And Networking">
-<meta property="og:title"     content="Uncomplicated Application-layer Vehicular Computing And Networking">
+<meta property="og:site_name" content="Open technology for real-time intravehicular distributed computing and communication based on modern networking standards">
+<meta property="og:title"     content="Open technology for real-time intravehicular distributed computing and communication based on modern networking standards">
```
Comment (thirtytwobits) [inline app/templates/base/head.html]: This is gobbledygook. How about just "Open technology for real-time communications in cyber-physical systems." ?

File: app/templates/consortium/content-column.html [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
+  Being an open project, OpenCyphal is critically dependent on direct contributions from its adopters.
+  This Consortium is a non-profit organization created to ensure the future of OpenCyphal and to provide a
   neutral collaboration platform for the stakeholders.
   The core objectives of the Consortium are:
 </p>
-<img src="/static/images/uavcan-conformity-mark.svg" width="160" height="160" class="img-fluid float-right p-3"
-     alt="UAVCAN Conformity Mark"
-     title="UAVCAN Conformity Mark">
+<img src="/static/images/cyphal-conformity-mark.svg" width="160" height="160" class="img-fluid float-right p-3"
+     alt="Cyphal Conformity Mark"
```
Comment (thirtytwobits) [inline app/templates/consortium/content-column.html:10]: There is no Cyphal conformity mark right now. Please be careful with this term. The community logo cannot be a conformity mark.

File: app/view/static.py [hunk 4]
```diff
@@ -13,17 +13,14 @@
...
     '/implementations*':                 '/',
     '/contact*':                         '/',
     '/uavcan*':                          '/',
-    '/gui_tool*':                        'https://forum.uavcan.org/c/app/yukon/14',
-    '/example*':                         'https://forum.uavcan.org/t/libcanard-examples-starters-tutorials/935',
-    # Old links to the implementations
-    '/*libuavcan*': 'https://github.com/UAVCAN/libuavcan',
-    '/*libcanard*': 'https://github.com/UAVCAN/libcanard',
-    '/*pyuavcan*':  'https://github.com/UAVCAN/pyuavcan',
+    '/gui_tool*':                        'https://forum.opencyphal.org/c/app/yukon/14',
```
Comment (thirtytwobits) [inline app/view/static.py]: Shouldn't this go to dronecan?

**Review by thirtytwobits — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pavel-kirienko) [reply README.md]: Fixed

File: app/templates/base/head.html (hunk 2 above)
Comment (pavel-kirienko) [reply app/templates/base/head.html]: Fixed

File: app/templates/consortium/content-column.html (hunk 3 above)
Comment (pavel-kirienko) [reply app/templates/consortium/content-column.html:10]: This part is commented out. Please use [uat.opencyphal.org:8080](http://uat.opencyphal.org:8080/) for review, not the source code.

File: app/view/static.py (hunk 4 above)
Comment (pavel-kirienko) [reply app/view/static.py]: My intention is to redirect this to Yukon one day. Come to think of it, why don't we redirect it to Yakut? It is already available. Fixing.

**Review by thirtytwobits — APPROVED** [review]

**Pull request merged**